src/subdomains/supporting/payout/entities/payout-order.entity.ts

```diff
@@ -132,4 +144,8 @@ export class PayoutOrder extends IEntity {
       amount: Util.round(this.payoutFeeAmount + this.preparationFeeAmount, 8),
     };
   }
+
+  get feeAmountChf(): number {
+    return this.preparationFeeAmountChf + this.payoutFeeAmountChf;
```

Comment:
@davidleomay is that true or only one amount?

Reply:
Is correct